src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import java.io.FileNotFoundException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws IOException {
+        System.out.println("Hello, my name is Duke! \nWhat can I do for you?");
+
+        ArrayList<Task> todoList;
+
+        Scanner myObj = new Scanner(System.in);
+
+
+
+        Path path = Paths.get("src","main","data");
+
+        try{
+            Files.createDirectories(path);
+        }catch (FileAlreadyExistsException ignored){}
```

Comment:
[Code Quality] Catching silently is almost always a bad thing to do. Consider how to improve this exception handling.